data_safe_haven/functions/strings.py

```diff
@@ -60,6 +63,18 @@ def sha256hash(input_string: str) -> str:
     return hashlib.sha256(input_string.encode("utf-8")).hexdigest()
 
 
+def time_as_string(hour: int, minute: int, timezone: str) -> str:
```

Comment:
Let's add some tests for this.

Reply:
Done. This helped me catch a bug in the function too (we were converting from UTC to local time rather than vice versa)!